### DISCUSSION TITLE
Scraper dati da sito web

### DISCUSSION THREAD

**Discussion by andbad** [indomus/forum#384] [HUB: Home Assistant]
Vorrei creare dei sensori recuperando dati da questo sito:
https://iss-mimic.github.io/Mimic/

Ho visto che in Home Assistant esiste la funzione "scrape", il problema è che (mi par di capire) il sito in questione non presenta i dati in modo statico ma dinamico. Infatti nel sorgente non vedo il "value" di un tag.
Ho creato un sensore così, ma il suo valore resta vuoto:
```
  - platform: scrape
    name: airlock000001
    resource: https://iss-mimic.github.io/Mimic/
    select: "#AIRLOCK000001"
```

Idee?

By(t)e

**Comment by snakuzzo** [comment]
Non ho avuto modo di provare in prima persona, nel tag con l'id che hai indicato non c'è nulla, quindi quello che ottieni è coerente

![image](https://user-images.githubusercontent.com/7205371/120391821-e652d700-c32f-11eb-87bc-7fdc05fed291.png)

**Reply by andbad** [reply]
Si, è esattamente quello che spiegavo. Se apri la pagina renderizzata invece il valore c'è. Probabilmente c'è uno script che lo genera e lo mostra.

By(t)e
